ninja-core/src/test/java/ninja/RouteBuilderImplTest.java

```diff
@@ -39,83 +41,69 @@
     Injector injector;
 
     @Test
-    public void testBasicGETRoute() {
-
+    public void basicGETRoute() {
```

Comment:
I think including "test" as a prefix adds verbosity.  I'd love to drop it, as well as prefer `assertThat` matchers as well.